ios/native/NativeProxy.mm

```diff
@@ -217,6 +217,25 @@ static id convertJSIValueToObjCObject(jsi::Runtime &runtime, const jsi::Value &v
     }
   };
 
+  auto configurePropsFunction = [reanimatedModule](
+                                    jsi::Runtime &rt, const jsi::Value &uiProps, const jsi::Value &nativeProps) {
+    NSMutableSet *usPropsSet = [[NSMutableSet alloc] init];
+    jsi::Array propsNames = uiProps.asObject(rt).asArray(rt);
+    for (int i = 0; i < propsNames.size(rt); i++) {
+      NSString *propName = @(propsNames.getValueAtIndex(rt, i).asString(rt).utf8(rt).c_str());
+      [usPropsSet addObject:propName];
+    }
+
+    NSMutableSet *nativePropsSet = [[NSMutableSet alloc] init];
+    propsNames = nativeProps.asObject(rt).asArray(rt);
+    for (int i = 0; i < propsNames.size(rt); i++) {
+      NSString *propName = @(propsNames.getValueAtIndex(rt, i).asString(rt).utf8(rt).c_str());
+      [nativePropsSet addObject:propName];
+    }
+
+    [reanimatedModule.nodesManager configureProps:usPropsSet uiProps:nativePropsSet];
```

Comment:
I'm not familiar enough with this to know if it's a bug or just confusing naming, but "uiPropsSet" not going to the "uiProps" argument isn't intuitively obvious.

In the code it replaced it was the opposite:

```objc
[nodesManager configureProps:[NSSet setWithArray:nativeProps] uiProps:[NSSet setWithArray:uiProps]];
```

Reply:
Yes, you are right 👍 I fixed it.